chore(#14) - Update vertx version to 3.4.2

miguelcnf opened the pull request.

miguelcnf committed:
chore(#14) - Update vertx version to 3.4.2

reistiago commented on src/main/java/com/statful/client/VertxMetricsImpl.java:
```diff
@@ -95,7 +90,7 @@ public boolean isEnabled() {
     @Override
     public void eventBusInitialized(final EventBus bus) {
-        this.customMetricsConsumer = new CustomMetricsConsumer(bus, this.getOrCreateSender(), statfulMetricsOptions);
+        new CustomMetricsConsumer(bus, this.getOrCreateSender(), statfulMetricsOptions);
```
We could continue to keep this reference mostly for a reminder that we should implement more method from the parent class. e.g.: close which should then stop that collector, along with the other collectors.

miguelcnf replied:
👍

miguelcnf committed:
refactor(#14) - Revert removing consumer instance field

This is mostly as a reminder that we should override the close method of the parent class which will in turn require us to stop the consumer.
